sass/extras.sass

```diff
@@ -0,0 +1,25 @@
+.title
+  font-size: $font-size-lg
+
+.subtitle
+  color: $gray
+  font-size: $font-size-xs
+  font-weight: lighter
+
+.header-line
```

Comment:
This should not be a separate element, but a pseudoelement linked to the title itself. So when you use the title, you don't have to remember about adding a separate component